skycoin/skywire
# [READY] Add configuration options for some apps in the manager

Did you run `make format && make check`?
Go code was not changed.

Fixes #268 

 Changes:	
-	This PR adds configuration options for skysocks and skysocks-client

How to test this PR:
Use the manager to check the app list of a visor. There should be a small gear button at the right of the skysocks and skysocks-client apps, which opens the configuration windows.

To test you can use the `make integration-run-proxy` command of the `skywire-services` repo.
